app/models/cart.rb

```diff
@@ -0,0 +1,15 @@
+class Cart < ActiveRecord::Base
+  belongs_to  :user
+  has_many    :order_items
```

Comment:
This is looking a lot better! What's your plan for items in a cart vs items in an order (like we were talking about with the receipt)? If you're planning to have items in an order as a separate class, this is going to be a confusing name for these.